service/models.py

```diff
@@ -113,18 +114,18 @@ def init_db(cls, app):
     @classmethod
     def all(cls):
         """ Returns all of the products in the database """
-        logger.info("Processing all products")
+        logger.info("Processing all Products")
         return cls.query.all()
 
     @classmethod
     def find(cls, by_id):
-        """ Finds a product by its ID """
+        """ Finds a product by it's ID """
```

Comment:
I changed this earlier, and the one below - 'its' is proper, which I know makes me a Grammar nerd/Nazi but...yeah.